replication: recovering from long WAL (#9963)
tarantool/tarantool: changelogs/unreleased/gh-9094-timeout-on-too-big-xlog-subscribe.md

## bugfix/replication

> * Fixed the bug that a replica could timeout on subscribe if the master would
>   have for that to open a big enough xlog file (gh-9094).

Reviewer: ```suggestion
* Fixed a bug when a replica could timeout on subscribe if the master had
  to open a big xlog file for that (gh-9094).
```
